server/routes/dashboardCollectionOverview.tsx

```diff
@@ -9,6 +9,12 @@ import { hostIsValid } from 'server/utils/routes';
 import { generateMetaComponents, renderToNodeStream } from 'server/utils/ssr';
 import { getCollectionOverview } from 'server/utils/queryHelpers';
 import { createUserScopeVisit } from 'server/userScopeVisit/queries';
+import { SubmissionWorkflow } from 'server/models';
+
+const getSubmissionWorkflow = async (collectionId: string): Promise<Number> => {
```

Comment:
Since the client just need to know whether or not a submission workflow exists for this collection, we could have this function return a boolean:

```ts
const collectionHasSubmissionWorkflow = async (collectionId: string) =>
  (await SubmissionWorkflow.count({ where: { collectionId } })) > 0;
```

Reply:
If you choose not to go that route, you can simplify this function by removing `async` and `await`, and make it a one-liner:

```ts
return SubmissionWorkflow.count({ where: { collectionId } }) as Promise<number>
```